hw/opentitan/ot_usbdev.c

```diff
@@ -591,7 +591,8 @@ static void ot_usbdev_update_vbus(OtUsbdevState *s)
     if (vbus_sense) {
         /* See BFM (bus_connect) */
 
-        /* If we end up in a non-disconnected state here, something is very wrong */
+        /* If we end up in a non-disconnected state here, something is very
+         * wrong */
```

Comment:
QEMU coding style requires block comments to use `/*` and `*/` on a line of their own:
```
/*
 * If we end up in a non-disconnected state here, something is very
 * wrong
 */
```